popsycle/synthetic.py

```diff
@@ -1727,7 +1732,7 @@ def _calc_blends(bigpatch, c, event_lbt, blend_rad):
 
     # Query ball against the existing (cached) tree.
     # NOTE: results is an array of lists.
-    results = kdtree_cache.query_ball_point(flatxyz1.T, r_kdt)
+    results = kdtree_cache.query_ball_point(flatxyz1.T.copy(order='C'), r_kdt)
```

Comment:
Why was this change necessary? What does it do?

Reply:
I kept getting an error that said "array is not C contiguous". After nearly a full day of trying everything I could this was the "easiest" fix. No one else seemed to be getting this issue. I couldn't figure it out for the life of me.

Reply:
You cannot do this. It has to be changed back because we do not know how it effects the results of the query.